examples/todoapp/lib/base/data_sources/remote/push_notification_data_source.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2023, Prime Holding JSC
+// https://www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:dio/dio.dart';
+import 'package:retrofit/http.dart';
+
+import '../../models/request_models/push_message_request_model.dart';
+import '../../models/request_models/push_notification_data_request_model.dart';
+
+part 'push_notification_data_source.g.dart';
+
+@RestApi()
+abstract class PushNotificationsDataSource {
```

Comment:
As agreed all generated by the rx_bloc_cli files, which are not used need to be removed

Reply:
Done